src/config/configDefaults.ts

```diff
@@ -0,0 +1,10 @@
+// changeable config variables
+export const SHOW_MUTATION_ASSESSOR = true;
+
+// variables below will be updated based on upper config variables
+export const ANNOTATION_QUERY_FIELDS = SHOW_MUTATION_ASSESSOR
+    ? ['annotation_summary', 'my_variant_info', 'mutation_assessor']
+    : ['annotation_summary', 'my_variant_info'];
+export const JSON_RESPONSE_QUERY_FIELDS = SHOW_MUTATION_ASSESSOR
+    ? '?fields=hotspots%2Cmy_variant_info%2Cptms%2Cannotation_summary%2Cmutation_assessor'
+    : '?fields=hotspots%2Cmy_variant_info%2Cptms%2Cannotation_summary';
```

Comment:
```suggestion
export const ANNOTATION_QUERY_FIELDS = ['hotspots', 'annotation_summary', 'my_variant_info', 'mutation_assessor']
```
Maybe something like that is a little cleaner. Feels a bit overkill to spread this over 3 variables, when all we really want to know is which fields we should query on genome nexus API. Disabling mutation assessor is just a matter of removing that query parameter

I added `hotspots` as well so everywhere we are querying the same sources. I guess that works?